### ISSUE TITLE
Wrong class for skip button

### ISSUE THREAD

**Issue by ink-spot** [xbsoftware/enjoyhint#7]
Hi there
the skip button is not initialized with the `enjoyhint_skip_btn` CSS class as it should. It uses the `enjoyhint_next_btn` class (line 263 of enjoyhint.js).

Thanks!

**Comment by asaskevich** [comment]
Is there any difference between `next` and `skip` styles? I think that now EnjoyHint works well and some style rules just deprecated. Thanks for the issue, we will check it in short time, :smile:

**Comment by ink-spot** [comment]
According to the CSS, the `skip` style is supposed to pin the button in the bottom-right corner.
Having the right class for the skip button would also allow to easily retrieve it through a jQuery selector, to customize it for exemple (even if a proper way to do that would be to give access to some customization options of this button in the library).

**Comment by asaskevich** [comment]
I fully agree about style customization. But about button's location.... Locations of the buttons are hardcoded inside library, because some users likes to see buttons near each other. In your opinion, maybe better to pass location of that button to the constructor?

**Comment by ink-spot** [comment]
I guess the ideal would be to allow the user to customize the buttons (skip and next) the way he wants (style, location, text).
I'm not a javascript expert so I wouldn't really know what would be the best way to do that, but it could be by setting custom classes and text value through the constructor config hash for exemple.

**Comment by blackazaru** [comment]
Hi, at the moment the button control is implemented. It will be possible either hide or display the buttons. The user will be able to do it himself. We will add it to github today or tomorrow.

**Comment by Amedrr** [comment]
Bug is still present.

Issue is in EnjoyHint::clear() function and appears when that.nextUserClass is undefined due to wrong async execution order.
> $(".enjoyhint_next_btn").removeClass(that.nextUserClass);

If that.nextUserClass === undefined, removeClass() cleans up entire element class and flushes next/skip buttons.

Following extra check makes it work perfectly:
> $(".enjoyhint_next_btn").removeClass(that.nextUserClass || '');
> $(".enjoyhint_skip_btn").removeClass(that.skipUserClass || '');

Feel free to make a commit.
Regards,
-- Alexander
